catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -380,14 +380,6 @@ define([
                 filters.push(typeFilter)
             }
 
-            var text = this.basicText.currentView.model.getValue()[0];
```

Comment:
I think this might be the wrong chunk of code to remove. See https://github.com/oconnormi/ddf/commit/017f56229c7b8e4f685edad3ec48a576f8a31cf5. I think we might have an issue where that change was accidentally reverted by a later commit that wasn't rebased.  I think maybe lines 353-356 should maybe be removed instead. Probably need to check with @andrewkfiedler on this.